File: src/main/java/ru/serdtsev/homemoney/moneyoper/MoneyOperStatus.java
package ru.serdtsev.homemoney.moneyoper;

public enum MoneyOperStatus {
  /**
   * в ожидании
   */
  pending,
  /**
   * ожидает повтора
   */
  recurrence,
  /**
   * выполнен
   */
  done,
  /**
   * выполнен (новый)
   */
  doneNew,
  /**
   * отменен
   */
  cancelled
}

File: src/main/kotlin/ru/serdtsev/homemoney/domain/model/moneyoper/MoneyOperStatus.kt
package ru.serdtsev.homemoney.domain.model.moneyoper

enum class MoneyOperStatus {
    New,
    /** Выполнен */
    Done,
    /** В ожидании */
    Pending,
    /** Ожидает повтора */
    Recurrence,
    /** Отменен, конечный статус */
    Cancelled,
    /** Шаблон */
    Template,
    /** Тренд */
    Trend
}